runtime/thunk/thunk.ts

```diff
@@ -313,7 +313,7 @@ async function doFindAndExecutePackFunction<T extends FormulaSpecification>({
           const validateParametersFormula =
             wrapMetadataFunction<ExecutionContext, boolean>(findValidateParametersFormula(manifest, formulaSpec));
           if (validateParametersFormula) {
-            return validateParametersFormula.execute(['', JSON.stringify({params: params as any, __brand: 'MetadataContext'})], executionContext);
+            return validateParametersFormula.execute(params as any, executionContext);
```

Comment:
this much more closely matches what we do with other params and in `executeFormulaOrSyncWithRawParams` we have this note which explains why this execute doesn't need three arguments when a metadataformula defines 3 arguments


```
case FormulaType.Metadata: {
      // Interestingly we don't need special handling for the formula context dict (the optional second arg
      // to an autocomplete metadata formula), because at execution time it gets passed as a serialized
      // JSON string anyway which is already parsed by the compiled pack definition.
      params = rawParams as ParamValues<ParamDefs>;
      // Default the search string (first arg) to an empty string.
      params[0] ??= '';
      break;
    }
```